File: src/main/java/io/github/bot/terminal/application/domains/db_connection/values/DbConnectionType.java
package io.github.bot.terminal.application.domains.db_connection.values;


public enum DbConnectionType {

    POSTGRES("PostgreSQL"),
    ORACLE("Oracle DB"),
    MYSQL("MySQL");

    private String label;

    DbConnectionType(String label) {
        this.label = label;
    }

    public String getLabel() {
        return label;
    }

    public interface Constants {
        String POSTGRES = "POSTGRES";
        String ORACLE = "ORACLE";
        String MYSQL = "MYSQL";
    }
}

File: src/main/kotlin/io/github/bot/terminal/application/domains/db_connection/values/DbConnectionType.kt
package io.github.bot.terminal.application.domains.db_connection.values

enum class DbConnectionType(val label: String) {
    POSTGRES("PostgreSQL"),
    ORACLE("Oracle DB"),
    MYSQL("MySQL");

    interface Constants {
        companion object {
            const val POSTGRES = "POSTGRES"
            const val ORACLE = "ORACLE"
            const val MYSQL = "MYSQL"
        }
    }

}